### DISCUSSION TITLE
vercel build failed with error:  Error: ENOENT: no such file or directory, lstat 'node_modules/next/dist/server/app-render.js'

### DISCUSSION THREAD

**Discussion by dasblitz** [vercel/community#2126] [Help]
### Summary

After upgrading to nextjs 13.3.0 my `vercel build`'s fail with:

`vercel build failed with error:  Error: ENOENT: no such file or directory, lstat 'correct-path-to-my/node_modules/next/dist/server/app-render.js'`

When I check what's in the `node_modules/next/dist/server/` there's a folder called `app-render` and inside there's a file `app-render.js` so the correct path should be: `/node_modules/next/dist/server/app-render/app-render.js`

I've tried removing node_modules and reinstalling using `yarn` but it did not help :(

Btw: I'm using NX for my monorepo setup. When running `yarn nx run build` it works as expected without any errors.

**Comment by amyegan** [comment]
Hi @dasblitz. Is the Build Command in your project settings also set to `yarn nx run build`, or is it using a different command?

**Reply by dasblitz** [reply]
No not entirely, in the vercel dashboard it's defined as: `npx nx build storefront --prod --skip-nx-cache`.

But if I run that locally in my terminal, there are no errors either

**Comment by dasblitz** [comment]
I've exploring a bit more and it seems related to the latest version of [NX](https://nx.dev/) `15.9.2` not really cooperating with next. In order to get the build working locally I had to add `transpilePackages: []` to my `next.config.js`. That seems to throw of the `vercel build` somehow.
